en/controllers.rst

```diff
@@ -670,8 +670,8 @@ Other Useful Methods
             array('limit' => 5, 'order' => 'Article.created DESC')
         );
 
-        $this->loadModel('User', 2);
-        $user = $this->User->read();
+        $this->loadModel('User');
```

Comment:
In this case it should stay like that as it explains the second parameter of the function ``loadModel``.